services/ingestion/etl/agri/csv/src/events/onParseCSV.js

```diff
@@ -1,71 +1,60 @@
-import path from 'path';
-import stream from 'stream';
 import AWS from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
 import parse from 'csv-parse';
 import transform from 'stream-transform';
 
-import transformRecord from '../lib/transform';
-
-// Destination bucket
-const { BUCKET } = process.env;
-
-export const handler = (event, context, callback) => {
-  /*
-   * Some checks here before going any further
-   */
-
-  // Only work on the first record
-  const snsRecord = event.Records ? event.Records[0] : undefined;
+import Logger from '../../../../../../logger/listener/src/lib/Logger';
 
-  // Was the lambda triggered correctly? Is the file extension supported? etc.
-  if (!snsRecord || snsRecord.EventSource !== 'aws:sns') {
-    return callback('Bad record');
-  }
-
-  /*
-   * Prepare file analysis
-   */
+// Import constants
+import { STATUS } from '../../../../../../storage/meta-index/src/events/onStatusReported';
```

Comment:
Can we move these constants to be part of the logger so all services can benefit extracting it from the `@eubfr/logger-listener`? If we make it a dependency as mentioned previously, of course ...